MyPiggyBank.Data/Model/CyclicOperation.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MyPiggyBank.Data.Model {
+    class CyclicOperation
+    {
+        public Guid CyclicOperationId { get; set; }
```

Comment:
Jak dla mnie, niepotrzebnie pisac CyclicOperationId. 
Wystarczy wpisać Id, bo to będzie Primary Key. Po co ta nadmiarowość. Poza tym na bazie będzie kłuć w oczy. Tak samo w tamtych klasach.